### PR TITLE
Add medicc2 to recipes

### PR DESCRIPTION
**Pull request by tlkaufmann**
Dear bioconda community,

my name is Tom Kaufmann, I am a PhD researcher at the Max Delbrück Center for Molecular Medicine. In the group of Dr. Roland Schwarz, we have further developed the [2014 MEDICC package](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003535) to include a range of new features relevant to the cancer evolution community.

MEDICC2 is a software for the estimation of Whole-genome doubling-aware copy number phylogenies for cancer evolution. For more information see the accompanying  paper [Whole-genome doubling-aware copy number phylogenies for cancer evolution with MEDICC2](https://www.biorxiv.org/content/10.1101/2021.02.28.433227v2).

----

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

### CODE REVIEW

**Comment by tlkaufmann** [comment]
@BiocondaBot please add label

File: recipes/medicc2/LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (druvus) [inline recipes/medicc2/LICENSE:1]: No need to include license file when it is included in compressed source file.

File: recipes/medicc2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler("cxx") }}
+  host:
```
Comment (druvus) [inline recipes/medicc2/meta.yaml:21]: Only python and pip should be necessary under host.

**Review by druvus — APPROVED** [review]

**Comment by tlkaufmann** [comment]
@BiocondaBot please update

**Comment by tlkaufmann** [comment]
@druvus, Thank you for your help Andreas!

**Comment by tlkaufmann** [comment]
@BiocondaBot please merge

**Pull request merged**